spyder/app/tests/conftest.py

```diff
@@ -297,6 +297,8 @@ def main_window(request, tmpdir, qtbot):
     # Set exclamation mark to True
     CONF.set('ipython_console', 'pdb_use_exclamation_mark', True)
 
+    CONF.set('editor', 'show_class_func_dropdown', True)
```

Comment:
Since there's only one test that checks the class/function dropdown, I think there's no need to enable it for all of them.

So, please move this line to be in the following `if` (placed below):

```python
if preload_complex_project:
    CONF.set('editor', 'show_class_func_dropdown', True)
    create_complex_project(tmpdir)
else:
    CONF.set('editor', 'show_class_func_dropdown', False)
    if not preload_project:
        CONF.set('project_explorer', 'current_project_path', None)
```